test/util/generateTestApp.js

```diff
@@ -5,14 +5,17 @@ const fse = require('fs-extra')
 const path = require('path')
 const util = require('util')
 
-module.exports = function (params, method) {
+module.exports = function (params, rooseveltPath, method) {
```

Comment:
Rather than making three arguments `rooseveltPath` and `method` should be members of an options object. That way in the future it will be easier to add more configuration possibilities. When you do this also rename `params` to `rooseveltParams` for clarity.